public/locations-list-map.js

```diff
@@ -399,86 +465,147 @@ function initMap(states) {
+    let stateCoords = { lat: stateLocationMapping.lat, lng: stateLocationMapping.lng },
+        stateZoom = stateLocationMapping.zoom;
+
+    // If this is an initialization (100% of the time at time of writing, trying to future proof) then also update our
+    // initial settings in order to retain this for reset link.
+    if (init) {
+      inital_center = stateCoords;
+      initial_zoom = stateZoom;
+    }
+
+    map.setCenter(stateCoords);
     // roughly zoom to focus on the state, hardcoded zooms to handle special case of Alaska
-    map.setZoom(stateLocationMapping.zoom);
+    map.setZoom(stateZoom);
   }
 }
 
-function centerMapToNearestMarkers(map, markers) {
-    // First check to see if the user will accept getting their location, if not, silently return
-    if (navigator.geolocation) {
-        navigator.geolocation.getCurrentPosition(function (position) {
-            var user_latlng = new google.maps.LatLng(position.coords.latitude, position.coords.longitude);
 
-            //Compute the distances of all markers from the user
-            var markerDistances = new Map(); // an associative array containing the marker referenced by the computed distance
-            var distances = []; // all the distances, so we can sort and then call markerDistances
-            for (const marker of markers) {
-                var distance = google.maps.geometry.spherical.computeDistanceBetween(marker.position, user_latlng);
 
-                // HACK: In the unlikely event that the exact same distance is computed, add one meter to the distance to give it a unique distance
-                // This could occur if a marker was added twice to the same location.
-                if (markerDistances.has(distance)) { distance = distance + 1; }
+/**
+ * Centers map at automatically detected coordinates using built in navigator.geolocation API.
+ */
+function centerMapToMarkersNearUser() {
+  // First check to see if the user will accept getting their location, if not, silently return
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function (position) {
+      // Use navigator provided lat/long coords to center map now.
+      centerMapToMarkersNearCoords(position.coords.latitude, position.coords.longitude);
+
+    }, function (err) {
+      // Hide the "User my location" link since we know that will not work.
+      $('#use-location').hide();
+
+    }, {
+      maximumAge: Infinity,
+      timeout: 10000
+    });
+  }
+}
 
-                markerDistances[distance] = marker;
 
-                distances.push(distance);
-            }
 
-            // sort the distances and set bounds to closest three
-            distances.sort((a, b) => a - b);
+/**
+ * Centers map around markers nearest to an arbitrary set of latitude/longitude coordinates.
+ */
+function centerMapToMarkersNearCoords(latitude, longitude) {
+  var latlng = new google.maps.LatLng(latitude, longitude);
```

Comment:
nit: replace var with const?

Reply:
Hard to realize, but not my code, this is part of refactor 😊 all my code uses let/const 😃 

Another major issue we gotta fix (see my todo at top) is the need to fix the insane indentation inconsistencies! This diff is so big that my indentation fixes made this diff even worse

Reply:
cc @graemer on that point about formatting.

Reply:
gonna ignore this for now, not the right branch